gui/src/main/java/org/jboss/as/console/client/shared/subsys/jca/ResourceAdapterPresenter.java

```diff
@@ -364,12 +367,15 @@ public void onCreateAdapter(final ResourceAdapter ra) {
         System.out.println(operation);
 
         dispatcher.execute(new DMRAction(operation), new SimpleCallback<DMRResponse>() {
-
-            @Override
-            public void onFailure(Throwable caught) {
-                super.onFailure(caught);
-                loadAdapter(false);
-            }
+        	
+			@Override
+			public void onFailure(Throwable caught) {
+				Log.error("Unknown error", caught);
+				Console.getMessageCenter().notify(
```

Comment:
Please use one of Console.error/info/debug instead. Using the message center directly is not recommened and could be removed in the future